### ISSUE TITLE
Failure on nix/mingw

### ISSUE THREAD

**Issue by ryobg** [dbry/WavPack#12]
Hi all,

this is more like informative. I have just git cloned on Debian, ran autogen.sh and configured with host x86_64-w64-mingw32 and all was fine until the script tried to link the cli tools. They were multiple undefined references to the functions in win32_unicode_support. I tried for an hour to fix this in more normal way, but I do not understand the autoconf so at the end I just added at EOF of wvunpack and wvgain include to the C file - it solved my problem.

I suppose that this file should be conditionally included when cross building.

Best regards.

**Comment by dbry** [comment]
Thanks for the heads-up on this! I installed mingw32 and reproduced your issue.

Just including the win32_unicode_support.c file for everyone would work because it's got a huge #ifdef _WIN32, but that's messy. I don't really understand that autoconf stuff either, but I have learned to cut & paste things I find until it works, and so I think I have this cleaned up (plus fixed that warning).

If you get a chance let me know if this fixes your case too and I'll close this issue. Thanks again!

-David
